### ISSUE TITLE
Inspection of multiple i386 images does not detect RHEL

### ISSUE THREAD

**Issue by katherine-black** [cloudigrade/houndigrade#105]
In GitLab by @mpierce on Jul 29, 2020, 17:21

## Summary

When I 

-  run one instance each of two different `i386` images that should both be identified as `RHEL`, 
-  then create a source and thereby a Cloud Meter account, 

Houndigrade's inspection process doesn't recognize my instances as RHEL

## Steps to Reproduce

1. Launch an instance each of the below two images
2. Create a Source, Cloud Meter account
3. Wait for inspection process to complete
4. Check facts returned by Cloud Meter about each instance

Image details:

```
Image(
        'ami-02c0a30866bcb43a8',
        'i386',
        't1.micro',
        'premium',
        'Red Hat Enterprise Linux Server',
        'i386-premium-cmTest1'
        ),
    Image(
        'ami-0e30b17e75753b989',
        'i386',
        't1.micro',
        'premium',
        'Red Hat Enterprise Linux Console',
        'i386-standard-cmTest2'
        ),
```

The above images each have a `/etc/rhsm/syspurpose/syspurpose.json` file with content similar to:

```
{
  "role": "Red Hat Enterprise Linux Server",
  "service_level_agreement": "premium",
  "usage": "test"
}
```

And they each have a `/etc/redhat-release` file containing:

```
Red Hat Enterprise Linux Server release 6.5 (Santiago)
```

## Expected Result

- Both instances are recognized as `RHEL` by Cloud Meter and show up in the data returned by the `Concurrent API`

## Actual Result

- Both instances are marked `RHEL: False` and `syspurpose: None`

## Additional context

This happens in the `QA` environment and hasn't been tested in other environments.
The AWS account used here is `DEV07`. 
I gathered details about images, instances and the account which can be found in attached files.

[888950.txt](/uploads/7593e90e8a72df2da0483adb6b85020b/888950.txt)

[888951.txt](/uploads/c7799b4599976db1f4b477c5b5a0c326/888951.txt)

**Comment by katherine-black** [comment]
In GitLab by @infinitewarp on Aug 3, 2020, 10:39

@mpierce Don't forget to check if this also fails when running only *one* of the images' instances.

**Comment by katherine-black** [comment]
In GitLab by @infinitewarp on Aug 3, 2020, 10:42

assigned to @infinitewarp

**Comment by katherine-black** [comment]
In GitLab by @mpierce on Aug 3, 2020, 18:18

What I've found today only adds to the (my) confusion. In running one instance of one of these images today, I've run into strange behavior/data. 

I haven't answered this particular question yet, but here's info related to I'm seeing that seems relevant to this overall context (but maybe not directly related)?

To sum-up, I triggered one instance of one i386 image via a test (create instance... add source... cloudmeter account). Somehow at time the intentional instance is created, three other instances of other images were also show as having been created in the Cloud Meter account but they do not appear in the AWS account (`DEV07`).

Maybe these other images are leftover from a previous test?? Except their created at timestamp is the same as the instance I created.

[details about expected image/instance](/uploads/1a5086ae82e46d4670981a38071e0ad0/890129.txt)

[details about bonus instances](/uploads/ad64eb23f1d8c9a674992079fdeadd1b/890130.txt)

¯\\_(ツ)_/¯

**Comment by katherine-black** [comment]
In GitLab by @mpierce on Aug 3, 2020, 18:57

Ok, and here are my findings of a single `i386` image/instance.

tldr - 

Expectation: `RHEL True`

Reality: `status: inspected` `RHEL: False` 

[details of single i386 instance](/uploads/c05f13db6c3b0cde3fb639259e9f9916/890131.txt)

**Comment by katherine-black** [comment]
In GitLab by @infinitewarp on Aug 6, 2020, 09:50

unassigned @infinitewarp

**Comment by katherine-black** [comment]
In GitLab by @infinitewarp on Aug 11, 2020, 09:52

marked this issue as related to cloudigrade#682

**Comment by katherine-black** [comment]
In GitLab by @infinitewarp on Aug 13, 2020, 13:30

assigned to @katherine-black

**Comment by katherine-black** [comment]
In GitLab by @pakamble on Sep 4, 2020, 09:32

assigned to @pakamble

**Comment by katherine-black** [comment]
In GitLab by @infinitewarp on Sep 11, 2020, 10:16

marked this issue as related to #51

**Comment by katherine-black** [comment]
In GitLab by @infinitewarp on Sep 11, 2020, 10:17

We just talked about this on a call, and since https://gitlab.com/cloudigrade/houndigrade/-/issues/51 should resolve the failed inspections for *some* of the i386 images, since the *general* use case appears to be good, we're calling this complete! :slight_smile:
